src/sinks/gcp/stackdriver_metrics.rs

```diff
@@ -279,8 +280,8 @@ mod tests {
         let mock_endpoint = spawn_blackhole_http_server(always_200_response).await;
 
         let config = StackdriverConfig::generate_config().to_string();
-        let mut config =
-            toml::from_str::<StackdriverConfig>(&config).expect("config should be valid");
+        let mut config = StackdriverConfig::deserialize(toml::de::ValueDeserializer::new(&config))
+            .expect("config should be valid");
```

Comment:
Serialization changed where `to_string()` returns the TOML map value with surrounding `{ }`s which doesn't work when using `toml::from_str`. Looking at https://docs.rs/toml/latest/toml/fn.from_str.html pointed to using [ValueDeserializer](https://docs.rs/toml/latest/toml/de/struct.ValueDeserializer.html) instead to deserialize this new value.